packages/runtime-core/src/customFormatter.ts

```diff
@@ -24,6 +24,7 @@ export function initCustomFormatter() {
   // custom formatter for Chrome
   // https://www.mattzeunert.com/2016/02/19/custom-chrome-devtools-object-formatters.html
   const formatter = {
+    __formater_key: 'vue-custom-formatter',
     header(obj: unknown) {
       // TODO also format ComponentPublicInstance & ctx.slots/attrs in setup
```

Comment:
While this PR works,  I prefer fixing the issue in the playground codebase.
Otherwise, we need some kind of versioning in the custom formatter.
Consider the scenario where we implemented the feature mentioned in this TODO comment in the future, and the page has loaded several versions of Vue; we would still need to add the latest custom formatter despite the presence of *another version of Vue custom formatter*.

Reply:
Do you mean that I should modify @vue/sfc-playground and find a way to ensure that the custom formatter is always up to date when using multiple versions of vue?

Reply:
I think just modifying the sfc-playground package is enough.

There are very few cases that we need to worry about too many devtoolFormatters so the deduplication might be overkill.